### PR TITLE
Iae 14329 Adding and updating comments

### PR DESCRIPTION
**Pull request by thomastighe**

### CODE REVIEW

File: src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.ts [hunk 1]
```diff
@@ -85,7 +115,7 @@ export class SamHierarchicalTreeComponent implements OnInit {
   }
 
   /**
-   * 
+   * Breadvrum selected
```
Comment (colinalford-gsa) [inline src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.ts]: Typo :)

File: src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.ts [hunk 2]
```diff
@@ -132,7 +162,7 @@ export class SamHierarchicalTreeComponent implements OnInit {
   }
 
   /**
-   * 
+   * Creats a new breadcrumb item and adds a new breadcrumb
```
Comment (colinalford-gsa) [inline src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.ts]: Typo :)

File: src/ui-kit/experimental/hierarchical/autocomplete/autocomplete.component.ts [hunk 3]
```diff
@@ -252,6 +251,10 @@ export class SamHierarchicalAutocompleteComponent {
     }
   }
 
+  /**
+   * Checks if the new search string matches the old search string
+   * @param searchString
```
Comment (colinalford-gsa) [inline src/ui-kit/experimental/hierarchical/autocomplete/autocomplete.component.ts]: If you don't have a description for the param, you can leave it out. Typescript should give us type information for the params.

File: src/ui-kit/experimental/hierarchical/hierarchical-tree-grid/hierarchical-tree-grid.component.ts [hunk 4]
```diff
@@ -42,60 +40,60 @@ export class SamHierarchicalTreeGridComponent implements OnInit {
...
 
   /**
-   * 
+   * Sort Directive
    */
   @ViewChild(SamSortDirective) sort: SamSortDirective;
 
   /**
-   * 
+   * constructor
```
Comment (colinalford-gsa) [inline src/ui-kit/experimental/hierarchical/hierarchical-tree-grid/hierarchical-tree-grid.component.ts]: Can you drop the comments for constructor, ngOnChanges, etc? The comment is duplicating the name of the method.

File: src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.ts (hunk 2 above)
Comment (thomastighe) [reply src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.ts]: CHanged

File: src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.ts (hunk 1 above)
Comment (thomastighe) [reply src/ui-kit/experimental/hierarchical/hierarchical-tree/hierarchical-tree.component.ts]: Fixed

**Pull request merged**